pkg/messageformat/parse.go

```diff
@@ -77,35 +94,33 @@ func (p *parser) parse() ([]interface{}, error) {
 	return tree, nil
 }
 
-func (p *parser) parseText() ([]interface{}, error) {
+// parseInsideCurly parses texts and expressions inside curly braces.
+func (p *parser) parseInsideCurly() ([]Node, error) {
```

Comment:
Perhaps would be a good idea to change function name -  `parseInsideCurly()` to something more generic?

Reply:
🤷